src/components/browse/components/FacetList/ExtendedDescriptionPopoverIcon.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+import React, { useMemo, useState, useRef } from 'react';
+import memoize from 'memoize-one';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import Fade from 'react-bootstrap/esm/Fade';
+
+import Popover from 'react-bootstrap/esm/Popover';
+
+
+export function FacetListPopoverController (props) {
+    const { children } = props;
+    // In this state we will store the entirety of the Popover JSX to display
+    // to allow flexibility in what to display in Popover contents
+    // in case we add more types of Popovers later.
+    const [ openPopoverID, setOpenPopoverID ] = useState(null);
+    const adjustedChildren = React.Children.map(children, function(child){
+        return React.cloneElement(child, { openPopoverID, setOpenPopoverID });
+    });
+    return (
+        <React.Fragment>
+            { adjustedChildren }
+
+        </React.Fragment>
+    );
+}
+
+
+/**
+ * @todo Potentially refactor to use ReactDOM.createPortal along w. semi-opaque dark background that will close this onClick.
+ * @todo Alternatively if we don't want the background, could attach event listener to window maybe..
+ */
+export const ExtendedDescriptionPopoverIcon = React.memo(function ExtendedDescriptionPopoverIcon (props) {
+    const { facet, fieldSchema = null, openPopover, setOpenPopover } = props;
+    const {
+        title: facetTitle,
+        field,
+        extended_description: facetExtendedDescription = null
+    } = facet;
+    const {
+        title: fieldTitle,
+        extended_description: fieldExtendedDescription = null
+    } = fieldSchema || {}; // May be null for 'calculated' or 'fake' fields.
+
+    //const [ isOpen, setOpen ] = useState(false);
```

Comment:
Can probably get rid of this now that some of those variables are in use elsewhere.